department-of-veterans-affairs/va.gov-cms
# VAGOV-3202 | VAGOV-3978 Linkit implementation

## What this does
- Sets linkit as the default link creator in wysiwyg and in link fields

## How to test
 - Go to `/node/715/edit`
 - In wyswyg field, select link icon, and begin typing - you should see a linkit dropdown
 - Add `List of link teasers` paragrpah
 - Begin begin typing in link field - you should see a linkit dropdown

## Definition of done
 - Above testing steps are successful

## Screenshots
![linkit-form](https://user-images.githubusercontent.com/2404547/62720763-b6a74200-b9d8-11e9-84f3-9597dc598b5c.png)
![should-show-drop-wysi](https://user-images.githubusercontent.com/2404547/62720764-b6a74200-b9d8-11e9-9f9b-92536006adfe.png)
![shoudl-drop-field](https://user-images.githubusercontent.com/2404547/62720765-b6a74200-b9d8-11e9-9f05-4507e8de61bc.png)
